src/doc/unstable-book/src/library-features/asm.md

```diff
@@ -345,6 +345,29 @@ The `h` modifier will emit the register name for the high byte of that register
 
 If you use a smaller data type (e.g. `u16`) with an operand and forget the use template modifiers, the compiler will emit a warning and suggest the correct modifier to use.
 
+## Memory address operands
+
+Sometimes assembly instructions require operands passed via memory addresses/memory locations.
+You have to manually use the memory address syntax specified by the respectively architectures.
+For example, in x86/x86_64 and intel assembly syntax, you could wrap inputs/outputs in `[]`
+to indicate they are memory operands.
+
+Old way:
+```rust,allow_fail
+# #![feature(llvm_asm)]
+# fn load_fpu_control_word(control: u16) {
+unsafe { llvm_asm!("fldcw $0" :: "m" (control) :: "volatile") }
+# }
+```
```

Comment:
I would move this down below the new way and prefix it with something like:

> Previously this would have been written with the deprecated `llvm_asm` like this: